packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -218,15 +220,23 @@ export class CodeFlowLogin {
           }
         })
         .catch(async (error) => {
-          CliCodeLogInstance.error("[Login] getTenantToken acquireTokenSilent : " + error.message);
-          const accountList = await this.msalTokenCache?.getAllAccounts();
-          for (let i=0;i<accountList!.length;++i) {
-            this.msalTokenCache?.removeAccount(accountList![i]);
+          if (error.message.indexOf(MFACode) >= 0) {
+            if (this.showMFA) {
+              console.log(colors.green(changeLoginTenantMessage));
+              this.showMFA = false;
+            }
+            return undefined;
+          } else {
+            CliCodeLogInstance.error("[Login] getTenantToken acquireTokenSilent : " + error.message);
+            const accountList = await this.msalTokenCache?.getAllAccounts();
+            for (let i=0;i<accountList!.length;++i) {
+              this.msalTokenCache?.removeAccount(accountList![i]);
+            }
+            this.config!.auth.authority = env.activeDirectoryEndpointUrl + tenantId;
```

Comment:
Why setting tenantId in this common login workflow?

Reply:
When login refresh token expires, try to use user used tenant id to login.